Renderer.js

```diff
@@ -0,0 +1,50 @@
+class Renderer {
```

Comment:
constructor here with `this.data = data` that was sent from main.js. main.js got the data from API manager